### PR TITLE
Feat: Liked video, History Video

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Feat: Liked video and History For Videos](https://github.com/pritam-kr/playme/commit/4e6cf09fba80a5d8567a21355d3202fb272d8993)[](https://github.com/pritam-kr)
- made a card for these two features
- user can jump into a single video page
- users can like videos and also they can see their watched videos
- user can delete liked video
- users can clear history

### Home
Screenshot (29)

### Liked Video
Screenshot (28)

### History Video
Screenshot (27)

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]
Design look's great :)

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -1,35 +1,75 @@
...
+                                {isVideo?.creator}
+                                <FaIcons.FaCheckCircle className="icons verified-icon" />
+                            </h3>
+                            <ul>
+                                <li
+                                    className="text-lg btn-likes"
+                                    onClick={() => likeVideoHandler()}
+                                >
+                                    <FaIcons.FaThumbsUp className="icons sidebar-icons" /> Like
+                                </li>
```
Comment (kshitijthatte) [inline src/Pages/SingleVideo/SingleVideo.jsx]: By performing a check if the video is already liked, you can add a toggle here and give an option to remove the like.

**Comment by poornima-alugubelly** [comment]
I think it'll be better if the user tries to like a video without signing in then user should be directed to sign in page, currently nothing happens

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (kshitijthatte) [inline src/Pages/SingleVideo/SingleVideo.jsx]: You can also add an alert, to notify the user that they need to log in to like the videos.

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 2]
```diff
@@ -1,35 +1,75 @@
...
+    const likeVideoHandler = () => {
+        saveLikedVideo(isVideo);
+    };
+
+    return (
+        getFilteredVideo && (
+            <div
+                className="single-videos-container main-container"
+                style={{ marginTop: "5rem" }}
+            >
```
Comment (poornima-alugubelly) [inline src/Pages/SingleVideo/SingleVideo.jsx:40]: if possible remove in-line styles

**Review by poornima-alugubelly — APPROVED** [review]

**Review by kshitijthatte — APPROVED** [review]
On page refresh on any other route than `/`, netlify renders 404 page. You'll need to add a `_redirects` file in the `public` folder to fix that. Apart from these, everything looks good.

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (pritam-kr) [reply src/Pages/SingleVideo/SingleVideo.jsx]: Its Done. Thank you

**Comment by pritam-kr** [comment]
> I think it'll be better if the user tries to like a video without signing in then the user should be directed to sign-in page, currently, nothing happens

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (pritam-kr) [reply src/Pages/SingleVideo/SingleVideo.jsx]: It's also Done Thank you

**Pull request merged**